### DISCUSSION TITLE
undefined reference to `absl::lts_20230802::log_internal::LogMessageFatal::~LogMessageFatal()'

### DISCUSSION THREAD

**Discussion by whutlz1994** [abseil/abseil-cpp#1567] [General]
### Describe the issue

**Building Abseil 20230802.0 with MinGW-w64 fails**

C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/include/google/protobuf/repeated_ptr_field.h:704:(.text$_ZN6google8protobuf8internal20Re
peatedPtrFieldBase13ClearNonEmptyINS0_16RepeatedPtrFieldIN4osi326LidarSensorView_ReflectionEE11TypeHandlerEEEvv[_ZN6google8protobuf8internal20RepeatedPtrFieldBase13ClearNonEmptyINS0_16RepeatedPtrFieldIN4osi32
6LidarSensorView_ReflectionEE11TypeHandlerEEEvv]+0xb5): **undefined reference to `absl::lts_20230802::log_internal::LogMessageFatal::~LogMessageFatal()'**
collect2.exe: error: ld returned 1 exit status
make[2]: *** [_deps/osi-build/CMakeFiles/open_simulation_interface.dir/build.make:129: _deps/osi-build/libopen_simulation_interface.dll] Error 1
make[1]: *** [CMakeFiles/Makefile2:293: _deps/osi-build/CMakeFiles/open_simulation_interface.dir/all] Error 2
make: *** [Makefile:136: all] Error 2

### Steps to reproduce the problem

cmake --build . --target install

### What version of Abseil are you using?

20230802.0

### What operating system and version are you using?

Windows 10

### What compiler and version are you using?

GCC  13.2.0

### What build system are you using?

CMake 3.27.7 + MSYS2

**Comment by whutlz1994** [comment]
Newcomer about  doing compile, please boss help me!

**Comment by whutlz1994** [comment]
I changed the version of protobuf from 24.3-1 to 21.12-1 in MINGW64(MSYS2), this issue fixed.
![Protobuf_version](https://github.com/abseil/abseil-cpp/assets/104348118/dea3ad36-dd87-4fab-98f5-913aa843b342)

**Reply by JinleiZhang0531** [reply]
you are right, i used [v21.12](https://github.com/protocolbuffers/protobuf/releases/tag/v21.12)  ,using MinGW64 compile success.
